subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -419,11 +419,11 @@ int aws_fota_mqtt_evt_handler(struct mqtt_client *const client,
 			LOG_ERR("MQTT PUBACK error %d", evt->result);
 			return 0;
 		}
-		return 0;
+		return 1;
 
 	case MQTT_EVT_SUBACK:
 		if (evt->result != 0) {
-			return 0;
+			return evt->result;
```

Comment:
This return value was not returned earlier, I guess this is an improvement compared to that?

Reply:
Yes as we now propagate the error so that it will be printed out.